src/createWidgetBase.ts

```diff
@@ -143,6 +149,24 @@ function manageDetachedChildren(instance: Widget<WidgetProperties>): void {
 	});
 }
 
+function bindFunctionProperties(instance: Widget<WidgetProperties>, properties: WidgetProperties) {
+	Object.keys(properties).forEach((propertyKey) => {
+		const func = properties[propertyKey];
```

Comment:
func probably violates our naming conventions, but I'll look the other way a I don't see a better option here.